platforms/php/prereqs/installPrereqs.sh

```diff
@@ -36,9 +36,13 @@ then
     add-apt-repository ppa:xapienz/curl34 -y
 fi
 
+# Set DEBIAN_FRONTEND environment variable
+export DEBIAN_FRONTEND=noninteractive
+
```

Comment:
why did we add this now only? are we hitting into any issue without this?

Reply:
No no.. started this PR a long time ago.. then added it.. and its fine to keep it.